### ISSUE TITLE
Width on 6 not working

### ISSUE THREAD

**Issue by ConsoleTVs** [uikit/uikit#2434]
When you set a width of 6 it does not work as expected:

```
<div uk-grid>
    <div class="uk-width-2-6">...</div>
    <div class="uk-width-4-6">...</div>
</div>
```

https://i.gyazo.com/9fb14fdc5245c7facf0d112844ec7095.png

Seems the only working ones (in terms of space) are:

```uk-width-1-6``` and ```uk-width-5-6```

**Comment by xspline** [comment]
+++ not working!

**Comment by zzseba78** [comment]
It´s not a bug, uikit remove alias ( redundant proportions ) from the grid, so uk-width-3-6 doesn´t exist anymore, you should write uk-width-1-2, and with uk-width-2-6 you should write uk-width-1-3... I'ts made to save some space. 
To me, it slow down the workflow, made you stop to think ( more that you are already thinking ).

**Comment by haydnjames** [comment]
I've noticed on the grid - where a width-*-* would duplicate another width declaration, it doesn't exist. 

So, because width-1-3 and width-2-6 would be the same, only width-1-3 exists as a valid tag. The same for width-2-3 and width-4-6. So I *think* the expectation is you need to mix width tags like this.

<div uk-grid>
    <div class="uk-width-1-6">...</div>
    <div class="uk-width-2-3">...</div>
    <div class="uk-width-1-6">...</div>
</div>

I guess this is to reduce the overall size of the CSS, but it will (and has) cause confusion.

**Comment by ConsoleTVs** [comment]
Indeed the documentation tells me that theese classes are available when they are not, and sometimes may cause confusion. I will edit the title for this if it's not a "real" issue.

**Comment by haydnjames** [comment]
I think the confusion has to outweigh the load gains here. You're only talking an extra 10-15 lines of CSS to add the duplicate/expected declarations in. For a doc that is already 9000+ lines, that addition is negligible.

**Comment by ConsoleTVs** [comment]
I agree few lines wont hurt anybody or at least, a graphical example in the docs (as well as the explanation, since it currently tell you they are valid) would be ideal.

**Comment by OndrejBakan** [comment]
I would rather sacrifice other classes (x-2, x-3, x-4, x-5) and have just the most specific (x-6) if the CSS size is an issue.

And as @ConsoleTVs is saying, documentation says these classes are available in whole range (1-x to (x-1)-x), it should be clearly stated they are not.

(btw I would love more columns, I think sixths are not enough for me).

**Comment by Heartfyre** [comment]
Just wanted to add that the documentation still states, that 1-6 to 5-6 is available. Took me about 30 minutes to find out, that it wasn't me doing something wrong, but the docs are wrong. Would be nice to remove or alter it, as I am pretty sure, that I am not the only one that ran into the issue, as seen here.

**Comment by ConsoleTVs** [comment]
@Heartfyre Yeah, should be done...
